pdc/apps/release/migrations/0002_auto_20150512_0719.py

```diff
@@ -53,6 +53,7 @@ def create_variant_types(apps, schema_editor):
         'optional',
         'addon',
         'layered-product',
+        'module',
```

Comment:
Not sure modifying previous migration is a good idea? I have a feeling this would only work on empty instances. But it wouldn't affect any already deployed version upgrade

Then again I am not an expert on migrations in Django :-)